build/release/common.sh

```diff
@@ -64,3 +74,10 @@ s3_upload() {
     # or ~/.aws/credentials and ~/.aws/config are configured
     aws s3 cp --only-show-errors ${filepath} s3://${RELEASE_S3_BUCKET}/${RELEASE_CHANNEL}/${RELEASE_VERSION}/${filename}
 }
+
+s3_promote_file() {
+    local filename=$1
+
+    aws s3 cp --only-show-errors s3://${RELEASE_S3_BUCKET}/master/${RELEASE_VERSION}/${filename} s3://${RELEASE_S3_BUCKET}/${RELEASE_CHANNEL}/${RELEASE_VERSION}/${filename}
```

Comment:
so promotions always happen from master, even when promoting to later channels such as beta and stable?